internal/app/init/pkg/system/services/kubelet.go

```diff
@@ -33,16 +33,34 @@ func (k *Kubelet) ID(data *userdata.UserData) string {
 
 // PreFunc implements the Service interface.
 func (k *Kubelet) PreFunc(data *userdata.UserData) error {
-	if err := os.MkdirAll("/etc/kubernetes/manifests", os.ModeDir); err != nil {
-		return fmt.Errorf("create /etc/kubernetes/manifests: %s", err.Error())
+	requiredMounts := []string{
+		// Commenting out the parent dir because of mkdirall
+		// "/dev",
+		"/dev/disk/by-path",
+		// Should we be more restrictive on what is exposed?
```

Comment:
Similar to my comments around the CNI, I think we should. This one may be difficult though, I don't know of a definitive list of paths under `/run` we might need :(.